src/operations/find_and_modify.ts

```diff
@@ -192,12 +192,7 @@ export class FindAndModifyOperation extends CommandOperation<Document> {
       ...this.cmdBase
     };
 
-    // Have we specified collation
-    try {
-      decorateWithCollation(cmd, coll, options);
-    } catch (err) {
-      return err;
-    }
+    decorateWithCollation(cmd, coll, options);
```

Comment:
I can revert this change, but without this the API doesn't error but returns an error.  That's not indicated by TS at all, so I figured it would be better to just throw.

Reply:
```js
    // Have we specified collation
    try {
      decorateWithCollation(cmd, coll, options);
    } catch (err) {
      return callback(err);
    }
```
I can see in 4.x this is the code, so it was an oversight in async/await conversion that this was left as a `return` rather than changed to a `throw`. Good catch!